lib/ecb_rates/application.rb

```diff
@@ -0,0 +1,28 @@
+require 'date'
+
+module EcbRates
+  class Application
```

Comment:
I think its great for usage simplificatiod and i added simple module method for that case. Otherwise  think its better to have content separated.  But moving all the errors to EcbRates root module might be also a good idea.